aws/aws.go

```diff
@@ -449,18 +467,22 @@ func deleteCFStacks(cfClients map[string]cfClient, rdsClients map[string]rdsClie
 	return errs
 }
 
-func disableDeleteProtection(rdsClient rdsClient, stackName string, dbInstanceId string, region string) error {
+func disableDeleteProtection(rdsClient rdsClient, stackName string, dbInstanceId string, region string) (bool, error) {
 	log.Infof("[AWS] Disabling DeletionProtection for DB instance: %s, region: %s", dbInstanceId, region)
+	dbExists := true
 
 	_, err := rdsClient.ModifyDBInstance(&rds.ModifyDBInstanceInput{
 		DBInstanceIdentifier: &dbInstanceId,
 		DeletionProtection:   aws.Bool(false),
 	})
-	if err != nil && !strings.Contains(err.Error(), "DB instance not found") {
-		log.Errorf("[AWS] Skipping stack %s delete because failed to disable DeletionProtection on db instance: %s, err: %s", stackName, dbInstanceId, err)
-		return err
+	if err != nil {
```

Comment:
For me this looks more readable, because the return value is straight forward, happy path is on the left as possible.
What do you think?
```
if strings.Contains(err.Error(), "DB instance not found") {
    return false, nil
}
log.Errorf("[AWS] Skipping stack %s delete because failed to disable DeletionProtection on db instance: %s, err: %s", stackName, dbInstanceId, err)
dbExists = false
```

Reply:
You are right, I have modified the check